ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -56,6 +56,12 @@ releases:
       version_bounds:
         lower: 4.15.0-0
         upper: 4.16.0-0
+  oldest-supported-z-rollback:
```

Comment:
We have dedicated ci yaml for rollback ci:
```
[openshift-tests-private]$ ls *rollback*
openshift-openshift-tests-private-release-4.12__amd64-rollback-nightly.yaml  openshift-openshift-tests-private-release-4.14__amd64-rollback-stable.yaml
openshift-openshift-tests-private-release-4.12__amd64-rollback-stable.yaml   openshift-openshift-tests-private-release-4.15__amd64-rollback-nightly.yaml
openshift-openshift-tests-private-release-4.13__amd64-rollback-nightly.yaml  openshift-openshift-tests-private-release-4.15__amd64-rollback-stable.yaml
openshift-openshift-tests-private-release-4.13__amd64-rollback-stable.yaml   openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml
openshift-openshift-tests-private-release-4.14__amd64-rollback-nightly.yaml  openshift-openshift-tests-private-release-4.16__amd64-rollback-nightly.yaml
```
We can add this job to there.

Reply:
Thanks, done.